messaging/src/main/java/org/frankframework/extensions/mqtt/MqttSender.java

```diff
@@ -83,18 +87,42 @@ public ParameterList getParameterList() {
 				super.start();
 			}
 
+			String topic = getTopic();
+			if (getParameterList() != null) {
+				IParameter topicParameter = getParameterList().findParameter(TOPIC_PARAMETER_NAME);
+
+				if (topicParameter != null) {
+					topic = topicParameter.getValue();
+				}
+			}
+
+			if (topic == null) {
+				throw new SenderException("Topic must not be null");
+			}
```

Comment:
Zou je in de `configure()` een test kunnen toevoegen dat als `topic == null` dat dan in ieder geval de topic-parameter aanwezig moet zijn in de parameter-lijst?

Reply:
Goeie! Zal ik doen